components/typography/anchor_link.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { globalTheme } from "../../theme";
+import { cx, css } from "react-emotion";
+
+class AnchorLink extends Component {
```

Comment:
Bethany and I discussed IRL and we'll implement the <Link> wrapper in this component and remove that functionality from the HeaderButton (so that will always return a button...)